.perltidyrc

```diff
@@ -0,0 +1,15 @@
+-pbp               # Start with Perl Best Practices
+--maximum-line-length=120
+--maximum-consecutive-blank-lines=2
+--indent-columns=2
+--opening-brace-always-on-right
+--cuddled-else
+--vertical-tightness=0
+--paren-tightness=2
```

Comment:
I think a value of `1` would more closely match @nelson-ferraz's original style. It will make the diff smaller, too.